assets/js/modules/analytics/components/common/UseSnippetSwitch.js

```diff
@@ -43,10 +45,10 @@ export default function UseSnippetSwitch() {
 	const onChange = useCallback( () => {
 		setUseSnippet( ! useSnippet );
 		trackEvent(
-			'analytics_setup',
-			useSnippet ? 'analytics_tag_enabled' : 'analytics_tag_disabled'
+			`${ viewContext }_analytics`,
+			useSnippet ? 'disable_ua_tag' : 'enable_ua_tag'
```

Comment:
See my other comment, as far as I can see, these were always the wrong way around.